docs/themes/thxvscode/layouts/partials/footer.html

```diff
@@ -43,10 +43,10 @@
                 <div class="copyright">
                     <a id="footer-microsoft-link" class="logo" href="https://www.microsoft.com">
                         <img class="microsoft-logo"
-                            src="https://fluidframework-docs-cdn.azureedge.net/static/images/microsoft-logo.png"
+                            src="https://storage.fluidframework.com/static/images/microsoft-logo.png"
                             height="20" alt="Microsoft homepage" />
                         <img class="microsoft-logo-inverted"
-                            src="https://fluidframework-docs-cdn.azureedge.net/static/images/microsoft-logo-inverted.png"
```

Comment:
These URLs are actually remnants from an old setup before we moved our Azure Front Door resource, which end up pointing to the old storage account as well. Removing the remnants of the old AFD here should let us delete the corresponding resource in Azure.